Source/Core/Common/Matrix.cpp

```diff
@@ -185,4 +215,74 @@ void Matrix44::Multiply(const Matrix44& a, const Vec4& vec, Vec4* result)
   result->data = MatrixMultiply<4, 4, 1>(a.data, vec.data);
 }
 
+// TODO: Perhaps this should return optional<>?
+Matrix44 Matrix44::Inverted() const
+{
+  float inv[16];
```

Comment:
`std::array`

It can then also use the `.size()` member function in the loop at the bottom of the function.

Reply:
:+1: